imports/ui/containers/dashboard.jsx

```diff
@@ -3,13 +3,24 @@ import { withTracker } from 'meteor/react-meteor-data';
 import Entries from '/imports/api/entries/entries.js';
 import Dashboard from '/imports/ui/components/dashboard/dashboard.jsx';
 
-const DashboardContainer = withTracker(() => {
+const categoryId = new ReactiveVar(3);
+const handleCategoryChange = (id) => categoryId.set(id);
+
+const DashboardContainer = withTracker((props) => {
   const entriesHandle = Meteor.subscribe('entries.board');
   const loading = !entriesHandle.ready();
-  const entries = Entries.find().fetch();
+  const currentUser = Meteor.user();
+  const currentUserName = Meteor.user().profile.name;
+  const categoryIdString = categoryId.get().toString();
+  const entries = categoryIdString === 'all' ? Entries.find().fetch() : Entries.find({userId: {$ne: currentUser._id}, "category.id": categoryIdString}).fetch()
```

Comment:
I'd implore to refactor this so that it fits within an 80 character line limit.

Reply:
Done! I broke this up into multiple lines.